be/src/service/internal_service.cpp

```diff
@@ -382,6 +386,26 @@ template <typename T>
 void PInternalServiceImplBase<T>::get_info(google::protobuf::RpcController* controller, const PProxyRequest* request,
                                            PProxyResult* response, google::protobuf::Closure* done) {
     ClosureGuard closure_guard(done);
+
+    GenericCountDownLatch<bthread::Mutex, bthread::ConditionVariable> latch(1);
+
+    if (!_async_thread_pool.try_offer([&]() { _get_info_impl(request, response, &latch); })) {
+        Status::ServiceUnavailable(
```

Comment:
BE's `routine_load_kafka_timeout_second` must recalculate as time between request finish and add to queue.
because FE also has `routine_load_kafka_timeout_second` which just little bit large then BE's config.

Reply:
Updated. The `get_info`'s timeout is now depend on the FE request and the interval duaring the queue waitting is counted. Besides, the timeout would be recaculated when the call consists several step.